frontend/projects/ui/src/app/pages/apps-routes/app-show/app-show.page.ts

```diff
@@ -53,6 +59,9 @@ export class AppShowPage {
     ),
   )
 
+  readonly currentMarketplace$: Observable<Marketplace> =
+    this.marketplaceService.getMarketplace().pipe(first())
```

Comment:
Why only first? What if this changes while I'm on the page?

Reply:
I thought it made sense to close the subscription here, as navigating away from and back to the component (to, say, change marketplaces) would cause this data to be "reloaded".

The only case I can think of where this would be problematic is if the user has two views of the UI open simultaneously, one where they are changing the marketplace and one where they are watching the service show page. In that case, using `first()` would disable the live update in the service show page. Seems like a rare case, but perhaps a better UX to remove this pipe since it's not strictly necessary.